scan-local.sh

```diff
@@ -22,11 +22,30 @@ docker run -d --name "$name" --volumes-from "$name-data" repo-info:local > /dev/
 
 echo '# `'"$image"'`'
 
+size="$(
+	docker inspect -f '{{ .VirtualSize }}' "$image" | awk '{
+		oneKb = 1000;
+		oneMb = 1000 * oneKb;
+		oneGb = 1000 * oneMb;
+		if ($1 >= oneGb) {
+			printf "~ %.2f Gb", $1 / oneGb
+		} else if ($1 >= oneMb) {
+			printf "~ %.2f Mb", $1 / oneMb
+		} else if ($1 >= oneKb) {
+			printf "~ %.2f Kb", $1 / oneKb
```

Comment:
I think we should swap these to KiB/MiB/GiB and use 1024 based math, since that is what `ls -h` gives:

```console
$ ls -l debian-config
-rw-r----- 1 sauron sauron 169868 Dec  1  2015 debian-config
$ ls -lh debian-config
-rw-r----- 1 sauron sauron 166K Dec  1  2015 debian-config
```

`df` lets you choose `-h` for 1024 and `-H` for 1000. `du` seems to be 1024 based.